### PR TITLE
Feature/profile link

### PR DESCRIPTION
**Pull request by elizabeth-jimenez**
[TM-1808](https://metaphase.atlassian.net/browse/TM-1808)
Dual Merge: [FE PR #921](https://github.com/MetaPhase-Consulting/State-TalentMAP/pull/921)

### CODE REVIEW

File: talentmap_api/user_profile/serializers.py [hunk 1]
```diff
@@ -116,6 +116,7 @@ class UserProfileSerializer(PrefetchedSerializer):
     cdo = StaticRepresentationField(read_only=True)
     is_cdo = serializers.ReadOnlyField()
     initials = serializers.ReadOnlyField()
+    employee_profile_url = serializers.ReadOnlyField()
```
Comment (mjoyce91) [inline talentmap_api/user_profile/serializers.py:119]: Is this needed?

Comment (elizabeth-jimenez) [reply talentmap_api/user_profile/serializers.py:119]: Sorry for the delay! Not sure lol, but I'll figure this out today :)

**Comment by scott062** [comment]
@mjoyce91 I saw you approved the FE, is this already good to go?

**Comment by mjoyce91** [comment]
Let's wait

**Comment by elizabeth-jimenez** [comment]
> @mjoyce91 I saw you approved the FE, is this already good to go?

@scott062 We got an update from WS this week. Mike told me yesterday that the link downloads a PDF. Um, I guess I will have to see how we want to handle that now, as we thought this would link to an employee's profile :/

**Pull request closed without merging**